package.json

```
@@ -24,4 +24,4 @@
     "lerna": "^5.0.0",
     "rimraf": "^3.0.2"
   }
-}
+}
```

[Formatting] Nit: unrelated change?

```suggestion
}

```